apps/supernova/src/components/silences/RecreateSilence.jsx

```diff
@@ -27,11 +27,16 @@ import constants from "../../constants"
 
 const validateForm = (values) => {
   const invalidItems = {}
-  if (values?.comment?.length <= 3) {
+  if (values?.comment?.length < 3) {
```

Comment:
Just a little comment, would be nice to extract this magic number into a constant please have look here https://stackoverflow.com/questions/47882/what-are-magic-numbers-and-why-do-some-consider-them-bad

Reply:
I do quite often constants. And not do it in a validation form where I just fixed the number that is equal to the error message. "please add at least 3 characters". Its also not global.